webapp/src/app.tsx

```diff
@@ -5545,6 +5545,7 @@ document.addEventListener("DOMContentLoaded", async () => {
     }
     pxt.perf.measureStart("setAppTarget");
     pkg.setupAppTarget((window as any).pxtTargetBundle);
+    appcache.init(() => theEditor.reloadEditor());
```

Comment:
Worth putting a comment here explaining the reason this needs to happen in this location. Something like `// don't place code above this line unless you know why it needs to happen before app refresh`

Reply:
Probably worth tagging this pr explicitly to give a trail in case it gets moved / git blame changes it for some reason too